crates/runtime-factors/src/lib.rs

```diff
@@ -100,6 +100,10 @@ pub struct TriggerAppArgs {
     /// To run from a file, prefix the filename with @ e.g. spin up --sqlite @migration.sql
     #[clap(long = "sqlite")]
     pub sqlite_statements: Vec<String>,
+
+    // Sets the maxmimum memory allocation limit for an instance.
```

Comment:
Units?

Reply:
memories

Reply:
In the words of my high school chemistry teacher:

"What are we talking here? Jelly beans, kumquats, frankfurters?"

Reply:
Given that the actual resolution here is pages it wouldn't be unreasonable for this to be KiB, though no one ever got fired for picking good old bytes...

Reply:
to be clear, I'm not asking for us to do extra work parsing kumquat/frankfurter units, bytes is fine for this level of advancedness, I just reckon we ought to say

Reply:
Updated to specify `in bytes`.